drivers/health_flexible_service/app/views/health_flexible_service/follow_ups/_form.haml

```diff
@@ -0,0 +1,44 @@
+- row_class = ''
+- col_class = ''
+- width_class = 'w-50'
+- if @pdf
+  - row_class = 'row'
+  - col_class = 'col-sm-6'
```

Comment:
This seems excessively complicated.  I believe you could just use `row` and `col-sm-6` in both situations.